pokemon-minisite/src/app/components/pokemon-list/pokemon-list.component.html

```diff
@@ -1,13 +1,16 @@
-<div>
-    <select class="select-dropdown" [(ngModel)]="selectedType">
-        <option value="all">All Types</option>
-        <option *ngFor="let type of types" [value]="type">{{ type }}</option>
-    </select>
-</div>
-<div>
-    <div><input [(ngModel)]="searchInput" (keydown.enter)="onSearchPokemon(searchInput)" placeholder="Search for Pokémon"></div>
+<div class="search">
+    <div>
```

Comment:
there are a lot of empty div , try use minimum div if there is no any effect

Reply:
done